src/python/pants/init/BUILD

```diff
@@ -17,8 +19,6 @@ target(
         "src/python/pants/backend/experimental/codegen/thrift/scrooge/java",
         "src/python/pants/backend/experimental/codegen/thrift/scrooge/scala",
         "src/python/pants/backend/experimental/debian",
-        "src/python/pants/backend/experimental/docker",
-        "src/python/pants/backend/experimental/docker/lint/hadolint",
```

Comment:
Bad change

Reply:
Are you referring to that you see both docker and docker/lint/hadolint here?
If so, pls review only the last commit, as the former is #14310

Reply:
Noticed the same remark in the other PR.. sorry for the confusion :)